dts/bindings/sensor/nxp,tmpsns.yaml

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2024 Polytech A/S
+# SPDX-License-Identifier: Apache-2.0
+
+description: NXP TMPSNS
+
+compatible: "nxp,tmpsns"
+
+include: base.yaml
```

Comment:
```suggestion
include: sensor-device.yaml
```

Reply:
Thanks @MaureenHelm, it should now be placed in the correct commit.